spark/dl/src/test/scala/com/intel/analytics/bigdl/nn/mkldnn/ReorderMemorySpec.scala

```diff
@@ -16,10 +16,76 @@
 package com.intel.analytics.bigdl.nn.mkldnn
 
 import com.intel.analytics.bigdl.mkl.Memory
+import com.intel.analytics.bigdl.nn.mkldnn.Phase.{InferencePhase, TrainingPhase}
 import com.intel.analytics.bigdl.tensor.Tensor
-import com.intel.analytics.bigdl.utils.BigDLSpecHelper
+import com.intel.analytics.bigdl.utils.Engine
+import org.scalatest.{BeforeAndAfter, FlatSpec, Matchers}
+
+class ReorderMemorySpec extends FlatSpec with Matchers with BeforeAndAfter {
+  "lenet5 with reorder" should "works no segment fault" in {
```

Comment:
what's this spec for? I think it should be in pr about updating to 0.17

Reply:
It can works on the two pr. In this case, it will tests run the dnn model in the same thread. The result should be no segment fault.